README.md

```diff
@@ -61,3 +61,22 @@ curl -X POST -d "grant_type=password&username=<user_name>&password=<password>&sc
 ```
 
 Now, this token will have read-only access to the API.
+
+
+## Testing
+
+1. Install [docker](http://docs.docker.com/installation/ubuntulinux/) and
+   [docker-compose](https://docs.docker.com/compose/install/)
+2. From the root project directory, run `docker-compose run test` (or `docker-compose
+   up`). You may need to use sudo depending on how you've configured Docker.
+
+If you encounter connection-refused errors, try re-running the test command;
+docker-compose will launch the postgres container first, but sometimes postgres doesn't
+become available quickly enough for Django to connect to it. Re-running will usually work.
```

Comment:
Wow. As you say, not a huge deal, wouldn't spend more than a few minutes on it. It's well documented here.